Cargo.toml

```diff
@@ -35,9 +35,13 @@ slog-async = "2.1"
 slog-scope = "~4.0"
 slog-term = "2.2"
 tempdir = "0.3"
-update-ssh-keys = "^0.4.0"
+tempfile = "^3.0.7"
+update-ssh-keys = { version = "^0.4.0", optional = true }
 users = "^0.8.1"
 
+[features]
+legacy = ["update-ssh-keys"]
```

Comment:
Features are usually additive. Do we want to call this something more specific like `merge-authorized-keys`? If we are going to add more CL-legacy features, we can always add an umbrella one called `cl-legacy`. Thoughts?

Reply:
Discussed out-of-band: this will likely ends up entangled in other things, such as project renaming. Let's stick to a monolithic `cl-legacy` to keep complexity low.